egs/aesrc2020/asr1/local/files/asr.dict

```diff
@@ -0,0 +1,1027 @@
+<unk> 1
```

Comment:
This seems like a bpe list. In the best case, we would like to make it from scratch or have some other places to store it.

Reply:
Thanks - I removed it. That file was not needed